bigchaindb/web/views/info.py

```diff
@@ -0,0 +1,25 @@
+"""This module provides the blueprint for some basic API endpoints.
+
+For more information please refer to the documentation in Apiary:
+ - http://docs.bigchaindb.apiary.io/
```

Comment:
I thought we were documenting in read the docs (https://bigchaindb.readthedocs.io/en/latest/drivers-clients/http-client-server-api.html)?